worlds/meritous/Regions.py

```diff
@@ -0,0 +1,59 @@
+# Copyright (c) 2022 FelicitusNeko
+#
+# This software is released under the MIT License.
+# https://opensource.org/licenses/MIT
+
+import typing
+from BaseClasses import MultiWorld, Region, Entrance, Location, RegionType
+from .Locations import MeritousLocation, alpha_store, beta_store, gamma_store, chest_store, special_store, location_table
+
+meritous_regions = ["Atlas Dome", "Chest rewards",
+                    "PSI Keys", "Meridian", "Ataraxia", "Merodach", "Endgame"]
+
+
+def create_regions(world: MultiWorld, player: int):
+    region_paid_stores = Region(
+        "Menu", RegionType.Generic, "Atlas Dome", player, world)
+    region_paid_stores.locations += [MeritousLocation(
+        player, loc_name, location_table[loc_name], region_paid_stores) for loc_name in [*alpha_store]]
```

Comment:
You don't need to do `[*dict]`, you can just do `for loc_name in alpha_store` and it will give you an iterator over the keys of the dictionary. Same for the other dicts.

Reply:
Done.